modules/wallet/persist.go

```diff
@@ -27,40 +29,45 @@ var (
 		Header:  "Wallet Seed",
 		Version: "0.4.0",
 	}
+
+	dbMetadata = persist.Metadata{
+		Header:  "Wallet Database",
+		Version: "1.0.0",
```

Comment:
We're going to be adding routes related to the wallet, namely some better restore-wallet routes, therefore the version needs to bump.